tests/support/helpers.lua

```diff
@@ -67,6 +67,13 @@ function M.get_today_date()
   return os.date('%Y-%m-%d')
 end
 
+-- Helper function to get yesterday's date string
+function M.get_yesterday_date()
```

Comment:
Extract both this helper and the production code to calculate the date into a function, so there is no duplication.